test/e2e/scenarios/add_cluster.go

```diff
@@ -0,0 +1,15 @@
+package scenarios
+
+import (
+	. "github.com/onsi/ginkgo"
+)
+
+var _ = PDescribe("Adding exisitng network policy to newly added cluster", func() {
+	PIt("Should implement exisitng network policy in newly added cluster", func() {
+		By("Create a new cluster")
```

Comment:
Is the idea to start with two clusters and then create a new one (i.e., cluster3)?

Reply:
Yes. It assumes that cluster 1 and 2 are already created with pods and network policy implemented. But I realised that's not a fair assumption and have added creating clusters 1 and 2 and network policy addition.